src/plugins/showHiddenChannels/index.tsx

```diff
@@ -492,6 +492,16 @@ export default definePlugin({
         }
     ],
 
+
+    fixPermCheck(originalPerms: number, channel: Channel) {
```

Comment:
okay it's actually bigint
```suggestion
    fixPermCheck(originalPerms: bigint, channel: Channel) {
```